plugins/catalog-react/src/components/EntityOwnerPicker/EntityOwnerPicker.tsx

```diff
@@ -101,16 +133,29 @@ export const EntityOwnerPicker = () => {
   }, [selectedOwners, updateFilters, availableOwners]);
 
   if (!availableOwners.length) return null;
+  if (error) throw error;
```

Comment:
Never throw in react components. You might want to render a `<ResponseErrorPanel error={error} />` if the async operation fails, or if space is tight, perhaps use the `errorApiRef` and post an error inside a useEffect that ends up as a callout at the top of the page.

Reply:
Looks like the mobile filter drawer has issues with 100% width components as children. Updated it to both use `errorApiRef` and show an error on the `Autocomplete`'s `TextInput`.